pilot/pkg/bootstrap/server.go

```diff
@@ -1198,3 +1203,46 @@ func (s *Server) initMeshHandlers() {
 		})
 	})
 }
+
+func (s *Server) initWorkloadTrustBundle() {
+	var err error
+	s.workloadTrustBundle = tb.NewTrustBundle(func() {
```

Comment:
Why do we have a global trust bundle *and* store it as a field in `Server`?

Seems like `workloadTrustBundle` could just be a local variable here - although probably preferred that its not global and instead we do store it in server (or return from this function) and pass it to pcds generator